protobuf/buf.md

```diff
@@ -0,0 +1,19 @@
+# Flagd build spec
+
+## This repository is managed by OpenFeature  
+
+
+This module contains the core types that developers can use for interacting with [flagd](https://github.com/open-feature/flagd).
+
+Internally flagd uses the connect protocol, meaning it is compatible with grpc interfaces. If your desired language has a supported plugin for generating connect stubs then it is reccomended to use these over grpc.
+
+The package contains a single `Service`, describing the 5 core `rpcs` for feature flag evaluation (`ResolveBoolean`, `ResolveString`, `ResolveFloat`, `ResolveInt` and `ResolveObject`) each with their type specific request and response objects(`ResolveXXXRequest` and `ResolveXXXResponse`).
+The final `rpc` on the `Service` is a streamed response named `EventStream`, this is used to pass internal events to the client, such as `configuration_change` and `provider_ready`.
+
+## Build options
+
+The core definitions are in the `schema.v1` package, and contains package name options for the following languages, as such these options may be excluded from build instructions:
+
+- Go: schema/service/v1       
```

Comment:
What do you think about taking this chance to update the go package name?

I think I'd like to see something more like the other examples. This is a nitpick though, I don't feel too strongly.

cc @skyerus

Reply:
We have another issue open for this [here](https://github.com/open-feature/schemas/issues/24). I tihnk we should hold off on making this change until after kubecon, as it will change the import paths for flagd and the go provider.